src/ui/src/components/core/content/CoreMetric.vue

```diff
@@ -133,7 +133,7 @@ const rootStyle = computed(() => {
 
 h2.value {
 	overflow: hidden;
-	text-overflow: ellipsis;
+	white-space: pre-wrap;
```

Comment:
You need to keep the `text-overflow`
```suggestion
	text-overflow: ellipsis;
	white-space: pre-wrap;
```

https://github.com/user-attachments/assets/730d22ff-4a67-42ae-9aa2-4965e44687ce